packages/core/src/user/UserService.ts

```diff
@@ -26,7 +26,7 @@ import {Availability, GenericMessage} from '@wireapp/protocol-messaging';
 import {AvailabilityType, BroadcastService} from '../broadcast/';
 import {ConnectionService} from '../connection';
 import {createId} from '../conversation/message/MessageBuilder';
-import {getPreKeyBundleMap} from '../messagingProtocols/proteus/Utility/PreKeyBundle';
+import {getPreKeyBundleMap} from '../messagingProtocols/proteus/Utility/PreKeyBundle/PreKeyBundle';
```

Comment:
```suggestion
import {getPreKeyBundleMap} from '../messagingProtocols/proteus/Utility/PreKeyBundle';
```

I think this should work since you have a `index.ts` in the root of PreKeyBundle